spark/src/main/scala/io/delta/sharing/spark/model.scala

```diff
@@ -96,7 +96,9 @@ private[sharing] case class Metadata(
     schemaString: String = null,
     configuration: Map[String, String] = Map.empty,
     partitionColumns: Seq[String] = Nil,
-    version: java.lang.Long = null) extends Action {
+    version: java.lang.Long = null,
+    size: java.lang.Long = null,
```

Comment:
could you also add these two new fields to PROTOCOL.md?

Reply:
Good catch, done.